modules/phase_field/include/kernels/NestKKSACBulkF.h

```diff
@@ -0,0 +1,81 @@
+//* This file is part of the MOOSE framework
+//* https://www.mooseframework.org
+//*
+//* All rights reserved, see COPYRIGHT for full restrictions
+//* https://github.com/idaholab/moose/blob/master/COPYRIGHT
+//*
+//* Licensed under LGPL 2.1, please see LICENSE for details
+//* https://www.gnu.org/licenses/lgpl-2.1.html
+
+#pragma once
+
+#include "KKSACBulkBase.h"
+
+// Forward Declarations
+
+/**
+ * KKSACBulkBase child class for the free energy difference term
+ * \f$ -\frac{dh}{d\eta}(F_a-F_b)+w\frac{dg}{d\eta} \f$
+ * in the the Allen-Cahn bulk residual.
+ *
+ * The non-linear variable for this Kernel is the order parameter 'eta'.
+ */
+// class KKSACBulkF : public Kernel
```

Comment:
```suggestion
```

Reply:
I would be happy to implement the KKS action at some point.